drivers/can/Kconfig.numaker

```diff
@@ -0,0 +1,13 @@
+# NuMaker CAN(-FD) driver configuration options
+
+# Copyright (c) 2023 Nuvoton Technology Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config CAN_NUMAKER
+	bool "Nuvoton NuMaker CAN-FD driver"
+	default y
+	select HAS_NUMAKER_CANFD
+	select CAN_MCAN
+	depends on DT_HAS_NUVOTON_NUMAKER_CANFD_ENABLED
+	help
+	  Enables Nuvoton NuMaker CAN-FD driver, using Bosch M_CAN as backend
```

Comment:
The purpose is to reveal, this driver is to utilize can_mcan instead of fully proprietary implementation.
If there is any concern, it's fine to remove this wording.

Reply:
I think you should remove it.